### PR TITLE
moves normalize to only be included with docs site to clean up dist bundle

### PR DESCRIPTION
**Pull request by kvuzet**
Moves normalize out of atlas.scss because it was getting included in app on import of a component, even though it already exists in the app. It is still rendered on the docs site but now I can export a cleaner dist.

### CODE REVIEW

**Review by demogar — APPROVED** [review]
Makes sense

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import './normalize.css';
```
Comment (demogar) [inline src/index.js:4]: however, should it be after the `index.css`?

Comment (kvuzet) [reply src/index.js:4]: oh good call, it should be before

Comment (kvuzet) [reply src/index.js:3]: fixed

**Pull request merged**